libgalois/src/PropertyManager.cpp

```diff
@@ -9,6 +9,15 @@ const std::string katana::PropertyManager::memory_category = "property";
 // Anchor manager vtable
 katana::Manager::~Manager() = default;
 
+namespace {
+
+std::string
+ToGB(uint64_t total) {
+  return fmt::format("{:.3f}", total / 1024.0 / 1024.0 / 1024.0);
```

Comment:
I think I have seen this function twice now. Could we put this in libsupport?

Reply:
I bet you always put your dishes in the dishwasher.

Reply:
Dishwasher? I wash my dishes with spit and dirt.